src/containers/edit-profile/notification-tab/NotificationTab.styles.ts

```diff
@@ -25,5 +32,8 @@ export const styles = {
   },
   switch: {
     ...switchStyle
+  },
+  updateProfileBtn: {
+    ...updateProfileBtn
   }
```

Comment:
You can just write field name as it is if they have the same name, don't need to write it two times
```
updateProfileBtn
```